packages/next/src/client/components/react-dev-overlay/ui/components/errors/dev-tools-indicator/dev-tools-info/user-preferences.tsx

```diff
@@ -165,6 +180,52 @@ export function UserPreferences({
           </div>
         </div>
       </div>
+      <h1 className="dev-tools-info-title">Development Server</h1>
+      <div className="preferences-container">
+        <div className="preference-section">
+          <div className="preference-header">
+            <label id="restart-dev-server">Restart Dev Server</label>
+            <p className="preference-description">
+              Restart the dev server to rebuild your application.
+            </p>
+          </div>
+          <div className="preference-control">
+            <button
+              aria-describedby="restart-dev-server"
+              name="restart-dev-server"
+              data-restart-dev-server
+              className="action-button"
+              onClick={handleRestartDevServer}
+            >
+              <span>Restart</span>
+            </button>
+          </div>
+        </div>
+      </div>
+      {process.env.__NEXT_TURBOPACK_PERSISTENT_CACHE ? (
+        <div className="preferences-container">
+          <div className="preference-section">
+            <div className="preference-header">
+              <label id="reset-bundler-cache">Reset Bundler Cache</label>
+              <p className="preference-description">
+                Reset the bundler cache to rebuild your application without
+                cache.
```

Comment:
Better copies are welcomed.

```suggestion
                Rebuild your application without the bundler cache.
```